File: project 3 - Array/php/LCM.php
<?php
if (isset($_GET['num1']) && isset($_GET['num2'])) {
    $num1 = intval($_GET['num1']);
    $num2 = intval($_GET['num2']);

    if ($num1 < $num2) {
        $smaller = $num1;
    } else {
        $smaller = $num2;
    }

    $gcd = 1;
    for ($i = 1; $i <= $smaller; $i++) {
        if ($num1 % $i == 0 && $num2 % $i == 0) {
            $gcd = $i;
        }
    }

    $lcm = ($num1 * $num2) / $gcd;

    echo "The Least Common Multiple (LCM) of $num1 and $num2 is: $lcm";
} else {
    echo "Please provide both num1 and num2 in the URL.";
}
?>

File: project 3 - Array/python/LCM.py
num1 = int(input("Enter the first number: "))
num2 = int(input("Enter the second number: "))

if num1 < num2:
    smaller = num1
else:
    smaller = num2

gcd = 1 
for i in range(1, smaller + 1):
    if num1 % i == 0 and num2 % i == 0:
        gcd = i 
lcm = (num1 * num2) // gcd

print(f"The Least Common Multiple (LCM) of {num1} and {num2} is: {lcm}")